src/main/java/MainClass.java

```diff
@@ -1,18 +1,25 @@
-import data.ScriptingProject;
 import gui.RootPane;
 import javafx.application.Application;
-import xml.XmlWriter;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class MainClass {
+    /**
+     * Add a logger to the application to keep track of all the things happening in the
+     * code. This logger has different levels and is able to distinguish between error
+     * levels.
+     */
+    static final Logger logger = LogManager.getLogger();
 
     /**
      *  @param args - the main arguments.
      */
     public static void main(String[] args) {
-        System.out.println("Welcome to the main class of goto-fail;");
+        logger.error("-------------------------------------------------------------------");
```

Comment:
I think info would be best. If I'm not mistaken the error level is only meant for exceptions, whereas this is general startup information. See [https://en.wikipedia.org/wiki/Log4j#Log4j_1_Log_level](url)